int64 DrillBitNumber
geometry_msgs/Pose NewTCP
---
string Result